src/main/java/edu/harvard/iq/dataverse/DatasetFieldValueValidator.java

```diff
@@ -168,7 +169,20 @@ public boolean isValid(DatasetFieldValue value, ConstraintValidatorContext conte
 
         if (fieldType.equals(FieldType.URL) && !lengthOnly) {
             try {
-                URL url = new URL(value.getValue());
+                
+                String[] schemes = {"http","https"};
+                UrlValidator urlValidator = new UrlValidator(schemes);
+                
+                String urlString = value.getValue();
+                URL url = new URL(urlString);
```

Comment:
FWIW: Not sure if this and the catch of a MalformedURLException is still needed. The urlValidator class tries to create a java URI and returns false if that gives a MalformedURLException

Reply:
I see, removed MalformedURLException.

Reply:
Indeed, removed `URL url = new URL(urlString)`.